inc/history.class.php

```diff
@@ -187,57 +187,71 @@ static function showCentralSpecificList($type) {
          return false;
       }
 
-      $groups     = implode("','", $_SESSION['glpigroups']);
-      $numrows    = 0;
-      $is_deleted = " `glpi_tickets`.`is_deleted` = 0 ";
+      $criteria = [
+         'SELECT' => ['glpi_tickets.id'],
+         'DISTINCT' => true,
+         'FROM' => 'glpi_tickets',
+         'LEFT JOIN' => [
+            'glpi_tickets_users' => [
+               'ON' => [
+                  'glpi_tickets' => 'id',
+                  'glpi_tickets_users' => 'tickets_id'
+               ]
+            ]
+         ],
+         'WHERE' => [
+            'glpi_tickets.is_deleted' => 0,
+         ],
+         'ORDER' => ['glpi_tickets.date_mod DESC']
+      ];
 
       if ($type == "notold") {
          $title = __("Tickets to follow (escalated)", "escalade");
          $status = CommonITILObject::INCOMING.", ".CommonITILObject::PLANNED.", ".
                    CommonITILObject::ASSIGNED.", ".CommonITILObject::WAITING;
```

Comment:
This has to be converted into an array of values.
```suggestion
         $status = [
             CommonITILObject::INCOMING,
             CommonITILObject::PLANNED,
             CommonITILObject::ASSIGNED,
             CommonITILObject::WAITING,
         ];
```